sdk/cosmos/azure-cosmos/azure/cosmos/_base.py

```diff
@@ -141,6 +141,18 @@ def GetHeaders(cosmos_client_connection,
     if options.get('offerThroughput'):
         headers[http_constants.HttpHeaders.OfferThroughput] = options['offerThroughput']
 
+    if options.get('contentType'):
```

Comment:
Shouldn't this have already been here for regular queries?

Reply:
it is already set directly in the query path : https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py#L2807

This will ensure it is set for other paths, like Queryplan